lib/utils/hierarchy_dataframe.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+import pandas as pd
+from typing import List
+
+
+def hierarchy_dataframe(metadata: pd.DataFrame, unique_id: str,
+    hierarchy_columns: List[str]) -> pd.DataFrame:
+    """
+    Given a ``pd.DataFrame`` `metadata`, returns a `pd.DataFrame` with one row
+    for every unique location hierarchy defined by the `unique_id` and
+    `hierarchy_columns` (e.g. region, country, and division). Looks for location
+    columns matching the original resolution name (e.g. 'region') and any
+    stubname of it (e.g. 'region_exposure').
+
+    Raises a :class:``KeyError`` if the given *unique_id* isn't in the
```

Comment:
I am definitely not getting the dostring documentation style right here. Which standard are we using for this repo?